app/controllers/admin_controller.rb

```diff
@@ -56,10 +59,10 @@ def check_permissions
   # Gets the user by the session and checks returns the boolean value in the database
   def check_is_admin
     user = User.find_by(username: session[:cas_user])
-    if user != nil
-     user.admin
+    if !user.nil?
```

Comment:
Whats the goal of this section here.  Maybe we should talk about this.  There are alot of times where we are checking things over and over again, but we should have a solid check somewhere before we begin that helps to sanitize, clean, verify, etc.  Maybe the checks are done in a separate class.